lib/virt_autotest/utils.pm

```diff
@@ -159,6 +159,7 @@ sub is_kvm_host {
 
 #retrun 1 if libvirt 9.0- is running which monolithic libvirtd is the default service
 sub is_monolithic_libvirtd {
+    record_info('WARNING', 'Libvirt package is not installed', result => 'fail') if (script_run('rpm -q libvirt-libs'));
```

Comment:
How will the result look like? When  is_monolithic_libvirtd is called, but no libvirt-libs is installed, how will openqa webui look like for the overall result and this record_info box?

Reply:
It will finally fail the test.
https://openqa.suse.de/tests/11758005#step/login_console/11

Reply:
```rpm -q libvirt-libs``` will fail on leap15.4 kvm host

Reply:
> I found libvirt-libs is dependency of libvirt. So it should be installed by default ?

Yes, It's the base package of libvirt.

Reply:
libvirt-libs installed by default when you choose to install kvm host pattern together with SLE Micro 5.5. I verified this on local machine.

Reply:
> It will finally fail the test. https://openqa.suse.de/tests/11758005#step/login_console/11

Looks good, thanks